pinot-common/src/main/java/org/apache/pinot/parsers/CompilerConstants.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.parsers;
+
+/**
+ * Class to host compiler related constants
+ */
+public class CompilerConstants {
+  public static final String AGGREGATION_FUNCTION_ARG_SEPARATOR = ":";
```

Comment:
this is only needed for distinct and it should go away after your change, right?

Reply:
The current approach is also based on concatenated arguments using a separator ":". Two other possibilities:

- Change AggregationInfo thrift object to pass multiple arguments in a much cleaner manner right from the parser stage.
- Change the way arguments are stored in the map that is already there in AggregationInfo. This doesn't require change to thrift definition. We can adopt a convention where multiple arguments are stored as individual KV pairs in the map that is already there in AggregationInfo. Right now there exists a single entry in this map. Key being "column" and value being column name (argument to aggr function)

Reply:
As @siddharthteotia pointed out, passing multiple args to aggregation function via AggregationInfo may end up creating a backward incompatible change. For now, I am piggy backing on the concatenation approach to pass multiple args. We can make a backward compatible cleanup separately, unless there is trivial fix that can be included in this PR.